website/static/css/pages/project-page.css

```diff
@@ -152,3 +152,7 @@ span#nodeDescriptionEditable{
     white-space: -pre-wrap;      /* Opera 4-6 */
     white-space: -o-pre-wrap;    /* Opera 7 */
 }
+
+#markdownRender {
```

Comment:
We try to avoid selecting CSS IDs in our CSS rules because they aren't reusable. Also, we already have a CSS helper class to hide UI that is rendered client-side, `.scripted`. Can you use that instead?